modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/UserOrderableServiceFlavor.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.servicetemplate;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.io.Serial;
+import java.util.List;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+/**
+ * Defines user orderable service flavor.
+ */
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class UserOrderableServiceFlavor extends ServiceFlavor {
```

Comment:
can you also please add a comment here to this class for future purpose?
`we need a separate data model to end user since the original flavor data model is used by ISV and CSP also have the pricing configuration info which should not be returned to the end user`